catalog/internals/webpack/webpack.prod.js

```diff
@@ -1,21 +1,16 @@
 const path = require('path')
-const TerserPlugin = require('terser-webpack-plugin')
+
 const CompressionPlugin = require('compression-webpack-plugin')
+const TerserPlugin = require('terser-webpack-plugin')
```

Comment:
It seems that TerserPlugin is the default, and there is no need to set it manually

https://webpack.js.org/configuration/optimization/#optimizationminimize

Reply:
we need to configure parallelism explicitly, bc otherwise it fails in CI